pycord/ext/gears/gear.py

```diff
@@ -20,22 +20,24 @@
 # SOFTWARE
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Type, TypeVar, Generic
+from types import SimpleNamespace
+from typing import TYPE_CHECKING, Any, Generic, Type, TypeVar
```

Comment:
Reverts out of scope change.
```suggestion
from typing import TYPE_CHECKING, Any, Generic, Type, TypeVar
```